data_processors/pipeline/domain/manops.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+"""manops domain module
+
+Domain models related to Manual of Operations (MANOPS)
+
+See domain package __init__.py doc string.
+See orchestration package __init__.py doc string.
+"""
+import json
+import os
+from typing import List
+from abc import ABC, abstractmethod
+from enum import Enum
+from libumccr import aws
+from data_portal.models.workflow import Workflow
+from data_processors.pipeline.domain.workflow import WorkflowType, WorkflowStatus
+from data_processors.pipeline.services.workflow_srv import get_succeeded_by_subject_id_and_workflow_type
+import boto3
+
+
+############################################################
+# Improvement: importing from libumccr when it is available
+# Ref: https://github.com/umccr/libumccr/commit/cbf781fbd271e63c17dd45bd73751bf4807c1f6c
+
+class LambdaInvocationType(Enum):
+    EVENT = 'Event'
+    REQUEST_RESPONSE = 'RequestResponse'
+    DRY_RUN = 'DryRun'
+
+
+############################################################
+
+
+class Report(Enum):
+    RNASUM = "rnasum"
+
+    @classmethod
+    def from_value(cls, value):
+        if value == cls.RNASUM.value:
+            return cls.RNASUM
+        else:
+            raise ValueError(f"No matching type found for {value}")
+
+    @staticmethod
+    def to_list():
+        return [e.value for e in Report]
+
+
+class ReportInterface(ABC):
+
+    @abstractmethod
+    def generate(self):
+        pass
+
+
+class RNAsumReport(ReportInterface):
+    """Triggering RNAsum report via lambda"""
+
+    def __init__(self):
+        super().__init__()
+        self.wfr_id = ""
+        self.dataset = ""
+
+    def add_dataset(self, dataset: str):
+        self.dataset = dataset
+
+    def add_workflow(self, wfr_id: str):
+        self.wfr_id = wfr_id
+
+    def add_workflow_from_subject(self, subject_id: str):
+        workflow_list: List[Workflow] = get_succeeded_by_subject_id_and_workflow_type(subject_id=subject_id,
+                                                                                      workflow_type=WorkflowType.UMCCRISE
+                                                                                      )
+        # Set if value exist
+        if len(workflow_list) > 0:
+            self.wfr_id = workflow_list[0].wfr_id
+
+    def generate(self):
+        fn_name = os.getenv('MANOPS_LAMBDA', 'data-portal-api-prod-manops')
```

Comment:
Can you point default to DEV i.e. `data-portal-api-dev-manops` instead here, pls.